firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -25,6 +25,17 @@ void moveMotor(int motor[], int steps, int dir)
 
 }
 
+void moveSpaces(int motor[], int spaces, int dir)
+{
+  int i;
+
+  digitalWrite(MOTOR_SLEEP, HIGH)
+  digitalWrite(MOTOR_RESET, HIGH)
+  digitalWrite(MOTOR_ENABLE, LOW)
+
+  for(i=0; i < )
```

Comment:
So this isn't bad cuz I won't merge the PR until the code compiles and everything, but try to avoid committing code that will break compilation or doesn't make sense.